aspnetcore/mvc/controllers/filters.md

```diff
@@ -374,7 +374,12 @@ The framework provides an abstract `ResultFilterAttribute` that you can subclass
 
 ### IAlwaysRunResultFilter and IAsyncAlwaysRunResultFilter
 
-The <xref:Microsoft.AspNetCore.Mvc.Filters.IAlwaysRunResultFilter> and <xref:Microsoft.AspNetCore.Mvc.Filters.IAsyncAlwaysRunResultFilter> interfaces declare an <xref:Microsoft.AspNetCore.Mvc.Filters.IResultFilter> implementation that runs for action results. The filter is applied to an action result unless an <xref:Microsoft.AspNetCore.Mvc.Filters.IExceptionFilter> or <xref:Microsoft.AspNetCore.Mvc.Filters.IAuthorizationFilter> applies and short-circuits the response.
+The <xref:Microsoft.AspNetCore.Mvc.Filters.IAlwaysRunResultFilter> and <xref:Microsoft.AspNetCore.Mvc.Filters.IAsyncAlwaysRunResultFilter> interfaces declare an <xref:Microsoft.AspNetCore.Mvc.Filters.IResultFilter> implementation that runs for all action results,  even if the response is short-circuited, except for:
```

Comment:
I like the version we just put in better because this says ... *runs for all* ... *even if* ... *except for*.

The one we put in says ... *that runs for action results* (period; it doesn't say "all"). Then ... 

> applied to an action result unless an \<xref:Microsoft.AspNetCore.Mvc.Filters.IExceptionFilter> or \<xref:Microsoft.AspNetCore.Mvc.Filters.IAuthorizationFilter> applies and short-circuits the response.

Reply:
The verbose part at the end is a respectable approach. Not commenting tho on the exact lingo there ... just saying that there's nothing wrong imo with explaining things out to make it super-clear.